### PR TITLE
Redesign link fields part 1

### PR DESCRIPTION
**Pull request by costebano**

### CODE REVIEW

File: UI/src/app/components/pages/mapping/mapping.component.scss [hunk 1]
```diff
@@ -131,3 +131,39 @@ footer {
     color:$cdm-grey-dark;
   }
 }
+
+::ng-deep .mat-tab-labels {
```
Comment (etokareva) [inline UI/src/app/components/pages/mapping/mapping.component.scss:135]: please get rid of ng-deep. It's deprecated in 10 angular. For future updates it'll cause double work.

To apply these styles  - they could be written in global scss files (preferable way to do this, e.g. create separate material styles file), or for now you can set  encapsulation: ViewEncapsulation.None

Comment (etokareva) [reply UI/src/app/components/pages/mapping/mapping.component.scss:135]: P.S. this comment is about all ng deep in this PR

File: UI/src/app/components/pages/mapping/mapping.component.scss [hunk 2]
```diff
@@ -131,3 +131,39 @@ footer {
...
+  display: block !important;
+}
+
+::ng-deep .mat-tab-group{
+  .mat-expansion-panel-header-title span {
+    color: #404040;
```
Comment (etokareva) [inline UI/src/app/components/pages/mapping/mapping.component.scss:141]: $cdm-grey-dark

File: UI/src/app/components/pages/mapping/mapping.component.scss [hunk 3]
```diff
@@ -131,3 +131,39 @@ footer {
...
+    font-weight: bold;
+    font-size: 14px;
+  }
+
+  .mat-ink-bar {
+    background-color: #066BBB !important;
```
Comment (etokareva) [inline UI/src/app/components/pages/mapping/mapping.component.scss:147]: $blue

File: UI/src/app/components/pages/mapping/mapping.component.scss [hunk 4]
```diff
@@ -131,3 +131,39 @@ footer {
...
+    margin-bottom: 10px;
+    border-bottom: none;
+
+    .mat-tab-label {
+      opacity: 1;
+      float: left;
+    }
+
+    .mat-tab-label-active .mat-tab-label-content {
+      color: #066BBB;
```
Comment (etokareva) [inline UI/src/app/components/pages/mapping/mapping.component.scss:161]: $blue

File: UI/src/app/components/panel/panel-source.component.scss [hunk 5]
```diff
@@ -21,14 +21,40 @@
...
+    ::ng-deep .linkSimilarItems {
       margin-left: auto;
       margin-right: 10px;
+
+      .mat-slide-toggle-bar {
+        width: 30px;
+      }
+
+      .mat-slide-toggle.mat-checked .mat-slide-toggle-bar {
+        background-color: #066BBB;
```
Comment (etokareva) [inline UI/src/app/components/panel/panel-source.component.scss:33]: $blue

File: UI/src/app/components/panel/panel-table/panel-table.component.scss [hunk 6]
```diff
@@ -81,10 +91,13 @@ table {
   }
 }
 
-// .row-has-a-link-true {
-//   border-right: 2px solid #80808033;
-//   background-color: #0000000d;
-// }
+.rowHasLink[ng-reflect-area="source"] {
+  background-color: #E5E5E5;
```
Comment (etokareva) [inline UI/src/app/components/panel/panel-table/panel-table.component.scss:95]: $cdm-grey-light

File: UI/src/app/components/panel/panel-target.component.scss [hunk 7]
```diff
@@ -21,14 +21,40 @@
...
+    ::ng-deep .linkSimilarItems {
       margin-left: auto;
       margin-right: 10px;
+
+      .mat-slide-toggle-bar {
+        width: 30px;
+      }
+
+      .mat-slide-toggle.mat-checked .mat-slide-toggle-bar {
+        background-color: #066BBB;
```
Comment (etokareva) [inline UI/src/app/components/panel/panel-target.component.scss:33]: $blue

File: UI/src/app/components/pages/mapping/mapping.component.scss (hunk 2 above)
Comment (costebano) [reply UI/src/app/components/pages/mapping/mapping.component.scss]: fixed

File: UI/src/app/components/pages/mapping/mapping.component.scss (hunk 3 above)
Comment (costebano) [reply UI/src/app/components/pages/mapping/mapping.component.scss]: fixed

File: UI/src/app/components/pages/mapping/mapping.component.scss (hunk 4 above)
Comment (costebano) [reply UI/src/app/components/pages/mapping/mapping.component.scss]: fixed

File: UI/src/app/components/panel/panel-source.component.scss (hunk 5 above)
Comment (costebano) [reply UI/src/app/components/panel/panel-source.component.scss]: fixed

File: UI/src/app/components/panel/panel-table/panel-table.component.scss (hunk 6 above)
Comment (costebano) [reply UI/src/app/components/panel/panel-table/panel-table.component.scss]: fixed

File: UI/src/app/components/panel/panel-target.component.scss (hunk 7 above)
Comment (costebano) [reply UI/src/app/components/panel/panel-target.component.scss]: fixed

File: UI/src/app/components/pages/mapping/mapping.component.scss (hunk 1 above)
Comment (costebano) [reply UI/src/app/components/pages/mapping/mapping.component.scss:135]: I will fix it in next PR

Comment (etokareva) [reply UI/src/app/components/pages/mapping/mapping.component.scss:135]: [Get rid of ng-deep](https://github.com/SoftwareCountry/CDMSouffleur/issues/57)

**Pull request merged**